core/tasks.py

```diff
@@ -39,7 +41,10 @@ def process_tv_webhook(table_id, view_id, record_id, event_type):
                 table_id, view_id, record_id, event_type))
             return
         elif event_type == 'AFTER_UPDATE':
-            record = getFullInvoiceData(record_id, view_id)
+            if manual_invoice_view_id == view_id:
```

Comment:
how do we know from the callback that it is the manual or normal invoice.

Reply:
fixed